src/explorer.api/Models/ExploreParams.cs

```diff
@@ -27,6 +29,8 @@ public string ApiUrl
         public string Table { get; set; } = string.Empty;
 
         [Required]
-        public ICollection<string> Columns { get; set; } = EmptyColumns;
+        public ImmutableArray<string> Columns { get; set; } = EmptyColumns;
```

Comment:
You can use `ImmutableArray<string>.Empty` here instead and remove the static readonly variable.

Reply:
fiexed with 92ecee6.